modules/manage/pages/kubernetes/k-manage-resources.adoc

```diff
@@ -336,7 +360,7 @@ helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --crea
 +
 ```bash
 helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
-  --set resources.cpu.cores=<number-of-cpu-cores> \
+  --set resources.requests.cpu=<number-of-cpu-cores> \
   --set resources.cpu.overprovisioned=true
```

Comment:
Ah, so same deal with the overprovisioned flag. It's now a CLI flag (like it is in redpanda) or it can be set in the redpanda.yaml which causes rpk to set the flag on you behalf.